pkg/baseline/packageIndex.json

```diff
@@ -45,8 +61,12 @@
             "AssemblyVersionInPackageVersion": {
                 "4.0.0.0": "4.0.0",
                 "4.0.1.0": "4.0.1",
-                "4.0.2.0": "4.3.1",
+                //"4.0.1.0": "4.0.2",
+                "4.0.2.0": "4.3.0",
+                //"4.0.2.0": "4.3.1",
```

Comment:
When assembly version is the same, but package version updated. Seems like make sense to keep the latest package version. This is applicable to all the other assembly-> package version mapping in this file

Reply:
It would seem so but if you take a look at PR #2955 @zhenlan made the same point but according to Eric this has some impact on package validation...

`The purpose of the version table in the package index is to tell the packaging infrastructure the minimum package version needed for a particular assembly version.`

Reply:
I see